src/components/BreadcrumbsBar/BreadcrumbItem/BreadcrumbItem.jsx

```diff
@@ -57,6 +57,7 @@ const BreadcrumbItem = ({
           text={text}
           icon={icon}
           isCurrent={isCurrent}
+          disabled={isDisabled || disabled}
```

Comment:
please replace with overrideDisabled which is purpose is to support disabled prop and also backward compatibility for deprecate isDisabled